perun-core/src/main/java/cz/metacentrum/perun/core/bl/GroupsManagerBl.java

```diff
@@ -89,11 +88,14 @@
 	 * @return newly created sub group with full group.Name with ":"
 	 * @throws GroupExistsException
 	 * @throws InternalErrorException
-	 * @throws GroupOperationsException
 	 * @throws GroupRelationNotAllowed
 	 * @throws GroupRelationAlreadyExists
+	 * @throws GroupNotExistsException
+	 * @throws AlreadyMemberException
+	 * @throws WrongReferenceAttributeValueException
+	 * @throws WrongAttributeValueException
 	 */
-	Group createGroup(PerunSession perunSession, Group parentGroup, Group group) throws GroupExistsException, InternalErrorException, GroupOperationsException, GroupRelationNotAllowed, GroupRelationAlreadyExists;
+	Group createGroup(PerunSession perunSession, Group parentGroup, Group group) throws GroupExistsException, InternalErrorException, GroupRelationNotAllowed, GroupRelationAlreadyExists, GroupNotExistsException, AlreadyMemberException, WrongReferenceAttributeValueException, WrongAttributeValueException;
```

Comment:
This method shouldn't throw ``GroupNotExistsException, AlreadyMemberException, WrongReferenceAttributeValueException, WrongAttributeValueException``. By it's implementation, it's thrown by ``createGroupUnion()`` which is fine, but in this case of creating new sub-group, there is no chance, there are any members (not already member / attributes exceptions) and also, if GroupNotExistsException would be thrown, it's some DB consistency error, so it should be wrapped as ``ConsistencyErrorException``. So please wrap it or silently skip such exceptions and log them.